common-pages/src/Setting.tsx

```diff
@@ -155,7 +155,7 @@ function Setting(props: IProps)  {
             dispatch(props.SettingsSlice.DBAction({verb: 'POST', record: editnewSetting}))
           if (conf && editNew === 'Edit')
             dispatch(props.SettingsSlice.DBAction({verb: 'PATCH', record: editnewSetting}))
-          if (!conf && isBtn)
+          if (!conf && (isBtn ?? false))
```

Comment:
This is a bug in `Modal` `isBtn` (and the 3rd parameter in `Callback`) should be `boolean` and not `?boolean` That would fix the issue here.

Reply:
I see, ill fix in `Modal` and revert this change.